docs/03 - Implementing the Anchor Server/Communication/Events Schema.yml

```diff
@@ -97,29 +97,36 @@ components:
             - transaction_created
             - transaction_status_changed
             - transaction_error
-          description:
-            transaction_created - a transaction was created via SEP31 ;
-            
-            transaction_status_changed - a transaction status change happened. The detail of the status change is described in the `status_change` field.
-            
-            transaction_error - error processing the transaction
+          description: |
+            The transaction event type. Can be one of the following:
+            - `transaction_created` - a transaction was created through the SEP endpoints.
+            - `transaction_status_changed` - the status of a transaction has changed. The detail of the status change is described in the `status_change` field.
+            - `transaction_error` - error processing the transaction.
+
         id:
           type: string
+          description: "The id of the transaction."
         sep:
           type: integer
+          description: "The SEP related to this TransactionEvent. Currently, only `31` is supported."
           enum: [31]
         kind:
           type: string
+          description: "The kind of transaction event. Currently, the only supported one is `receive`."
           enum: ["receive"]
         status:
           type: string
+          description: |
+            The status of the transaction according with the latest Platform update. Currently, we're supporting all [SEP-31 states](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#get-transaction).
           enum: [
             "pending_sender",
             "pending_stellar",
             "pending_customer_info_update",
+            "pending_transaction_info_update",
```

Comment:
I didn't add this because the platform doesn't support custom per-transaction fields, and the only time `pending_transaction_info_update` is used is when those fields need updating.

Reply:
Removed in https://github.com/stellar/java-stellar-anchor-sdk/pull/412/commits/df6d62803935be72225784b835eabd1797eacfab.